src/WireMock.Net.Abstractions/Admin/Mappings/XmlNamespace.cs

```diff
@@ -0,0 +1,20 @@
+namespace WireMock.Admin.Mappings;
+
+/// <summary>
+/// Defines an xml namespace consisting of prefix and uri.
+/// <example>xmlns:i="http://www.w3.org/2001/XMLSchema-instance"</example>
+/// </summary>
```

Comment:
add `[FluentBuilder.AutoGenerateBuilder]`

Reply:
Ah, ok. I'll do that!

Reply:
Done